test/float/smack_code_annot.c

```diff
@@ -4,7 +4,7 @@
 
 double f16_to_f32(float x) {
   double ret = __VERIFIER_nondet_double();
-  __SMACK_code("@ := ftd(RNE, @f);", ret, x);
+  __SMACK_code("@ := $fpext.bvfloat.bvdouble(RNE, @f);", ret, x);
```

Comment:
Why do you have bvfloat into bvdouble if the function is `f16_to_f32`?
It seems to be that the types do not match the function name.

Reply:
Good catch. Fixed.